tutorial/transport/py_transport.ipynb

```diff
@@ -223,22 +207,36 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# capacity of plant i in cases                                                                                                                                                           \n",
-    "# add parameter elements one-by-one (string and value)                                                                                                                                   \n",
-    "scen.init_par(\"a\", idx_sets=\"i\")\n",
-    "scen.add_par(\"a\", \"seattle\", 350, \"cases\")\n",
-    "scen.add_par(\"a\", \"san-diego\", 600, \"cases\")\n",
+    "import pandas as pd\n",
     "\n",
-    "# demand at market j in cases                                                                                                                                                            \n",
-    "# add parameter elements as dataframe (with index names)                                                                                                                                 \n",
-    "scen.init_par(\"b\", idx_sets=\"j\")\n",
-    "b_data = [\n",
-    "    {'j': \"new-york\", 'value': 325, 'unit': \"cases\"},\n",
-    "    {'j': \"chicago\",  'value': 300, 'unit': \"cases\"},\n",
-    "    {'j': \"topeka\",   'value': 275, 'unit': \"cases\"}\n",
-    "]\n",
-    "b = pd.DataFrame(b_data)\n",
-    "scen.add_par(\"b\", b)"
+    "from ixmp4.core import Unit\n",
+    "\n",
+    "# Only needed once for each mp\n",
+    "try:\n",
+    "    cases = mp.units.get(\"cases\")\n",
+    "except Unit.NotFound:\n",
+    "    cases = mp.units.create(\"cases\")\n",
+    "\n",
+    "# capacity of plant i in cases\n",
+    "# add parameter data as a dict\n",
+    "a = run.optimization.parameters.create(name=\"a\", constrained_to_indexsets=[\"i\"])\n",
+    "a_data = {\n",
+    "    \"i\": [\"seattle\", \"san-diego\"],\n",
+    "    \"values\": [350, 600],\n",
+    "    \"units\": [cases.name, cases.name],\n",
```

Comment:
Yes, because they describes lists of values and units. The alternative would be to have a column called `value`, which actually represents multiple values.

A very similar issue could be discussed for `levels` and `marginals` for `Variable` and `Equation`. I've applied the same reasoning there.

Reply:
I see that you applied this reasoning consistently, but it is a departure from the **ixmp** API. It is also quite clear that the column "value" has all values, so in the interest of least disruption for existing users, I'd keep the ixmp convention.

Reply:
Okay, I will change to `value` and `unit` here. For consistency, we should then also probably change to `indexset.element`, `variable.level` etc, which I don't want to do here because the purpose of this PR should be to bring in the tutorial, not to change unmerged DB content.

Reply:
Ok to do this in other PRs, sorry that I didn’t catch it earlier. But we don’t want to ask colleagues to update all their scripts for MESSAGE data pre- or postprocessing. Maybe the item attributes can stay as is, but the tabular representation should be consistent with **ixmp**.

Reply:
Turns out: I can't even do that for `parameter.values` and `parameter.units` as these column names are also required (explicitly in plural form). But let's create a new issue from this comment and keep this in mind when reviewing the other PRs. I would suggest merging the other PRs without fixing this (requires less rework of this tutorial) and then have a separate clean up PR for this issue fixing at the DB level and the tutorial.

To be clear: at the core level, you want the `parameter.values` etc to stay plural form?

Reply:
Continued in #102.